### PR TITLE
[NCL-5109] Change WebSocket URL

### PR DESCRIPTION
**Pull request by thescouser89**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/pnc/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

**Comment by thescouser89** [comment]
retest this please

**Comment by thescouser89** [comment]
retest this please

**Comment by thescouser89** [comment]
retest this please

**Comment by thescouser89** [comment]
retest this please

**Comment by thescouser89** [comment]
retest this please

File: integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/notifications/ProcessProgressNotificationTest.java [hunk 1]
```diff
@@ -126,7 +121,7 @@ public void shouldSubscribeToProcessUpdatesNotification() throws Exception {
         //then
         logger.info("Received: " + notificationCollector.getMessages().get(0));
 
-        assertThat(notificationCollector.getMessages().get(0)).startsWith("{\"oldStatus\":\"NEW\",\"build\":{\"id\":\"1\",\"status\":\"SUCCESS\",");
+        assertTrue(notificationCollector.getMessages().get(0).startsWith("{\"oldStatus\":\"NEW\",\"build\":{\"id\":\"1\",\"status\":\"SUCCESS\","));
```
Comment (janinko) [inline integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/notifications/ProcessProgressNotificationTest.java]: why did you changed it from assetj to just junit?

File: rest-new/pom.xml [hunk 2]
```diff
@@ -56,6 +56,16 @@
         </dependency>
 
         <!-- REST dependencies -->
+        <dependency>
+            <groupId>org.jboss.pnc</groupId>
+            <artifactId>notifications</artifactId>
+            <scope>provided</scope>
+        </dependency>
```
Comment (janinko) [inline rest-new/pom.xml]: This should be among the Project dependencies

File: integration-test/pom.xml [hunk 3]
```diff
@@ -68,6 +68,12 @@
       <scope>test</scope>
       <classifier>classes</classifier>
     </dependency>
+    <dependency>
+      <groupId>org.jboss.pnc</groupId>
+      <artifactId>rest-new</artifactId>
+      <scope>test</scope>
+      <classifier>classes</classifier>
+    </dependency>
```
Comment (janinko) [inline integration-test/pom.xml]: why is this needed? What does it do?

File: integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/notifications/ProcessProgressNotificationTest.java (hunk 1 above)
Comment (thescouser89) [reply integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/notifications/ProcessProgressNotificationTest.java]: There was a classloading issue with assertj, which is why I just replaced it with JUnit

File: integration-test/pom.xml (hunk 3 above)
Comment (thescouser89) [reply integration-test/pom.xml]: It bundles all the rest-new classes, so that my test classes can have their @Injected objects working. Otherwise I suffered from NoClassFoundException IIRC

File: integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/notifications/ProcessProgressNotificationTest.java (hunk 1 above)
Comment (janinko) [reply integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/notifications/ProcessProgressNotificationTest.java]: only in this one file?

Comment (thescouser89) [reply integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/notifications/ProcessProgressNotificationTest.java]: Yup. It's also IIRC the only file that uses it.

File: rest-new/pom.xml (hunk 2 above)
Comment (thescouser89) [reply rest-new/pom.xml]: OK

Comment (thescouser89) [reply rest-new/pom.xml]: It doesn't seem to work without it. See Jenkins job: 4154

**Comment by janinko** [comment]
I had trouble compiling it both on Java 8 and java 11.

**Comment by thescouser89** [comment]
Eugh will rebase that PR. The dangers of PRs that linger for too long

**Comment by thescouser89** [comment]
@janinko  Attaboy! I had to add the jakarta xml bind dep to the notifications module since the `javax.xml.bind` stuff got removed in Java SE 11

File: rest-new/pom.xml (hunk 2 above)
Comment (janinko) [reply rest-new/pom.xml]: I just meant move this dependency 2 lines up :)
